src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -80,7 +80,7 @@
   <!--
     The GetTargetPathWithTargetPlatformMoniker target uses a BeforeTargets so the only way to disable it is to override it with an empty target.
   -->
-  <Target Name="GetTargetPathWithTargetPlatformMoniker" />
+  <!--<Target Name="GetTargetPathWithTargetPlatformMoniker" />-->
```

Comment:
nitpick: why comment this out, rather than remove it entirely?

Reply:
Yes, I agree. It commented it out to try if that resolved my issue, but forgot to remove it after that. Will do that now.